### PR TITLE
Added docker file and set up github actions with docker

### LINKED ISSUE #2: Basic scraper structure
- entrypoint with basic params: commons, lang, S3, low-quality, no-external-links
- scraper
- templates
- dependencies
- Dockerfile
- codefactor
- black CI
- docker Workflow

### PR DESCRIPTION
**Pull request by ElenaKusevska**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM python:3.8-slim
+LABEL org.opencontainers.image.source https://github.com/openzim/kolibri2zim
```
Comment (rgaudin) [inline Dockerfile]: repo name is `kolibri`

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - feature_docker
```
Comment (rgaudin) [inline .github/workflows/docker.yml]: Change to master. We'll ensure it's working fine once merged

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (rgaudin) [inline Dockerfile:3]: Please add locale and timezone

```Dockerfile
RUN echo "UTC" >  /etc/timezone
ENV TZ "UTC"

# setup locale
RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && locale-gen
ENV LANG en_US.UTF-8
ENV LANGUAGE en_US:en
ENV LC_ALL en_US.UTF-8
```
```

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+# Install necessary packages
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends locales-all unzip wget ffmpeg \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# TEMP
+RUN wget -L http://tmp.kiwix.org/wheels/libzim-1.0.0.dev1-cp38-cp38-manylinux1_x86_64.whl http://tmp.kiwix.org/wheels/zimscraperlib-1.4.0.dev1-py3-none-any.whl
```
Comment (rgaudin) [inline Dockerfile]: Just do as for sotoki

```Dockerfile
# TEMP: install pylibzim and scraperlib through built wheels (until release)
RUN wget --progress=dot:giga http://tmp.kiwix.org/wheels/libzim-1.0.0.dev1-cp38-cp38-manylinux1_x86_64.whl \
    && wget --progress=dot:giga http://tmp.kiwix.org/wheels/zimscraperlib-1.4.0.dev2-py3-none-any.whl \
    && pip install --no-cache-dir ./*.whl && rm -f ./*.whl
```

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+RUN pip3 install --no-cache-dir libzim-1.0.0.dev1-cp38-cp38-manylinux1_x86_64.whl && rm libzim-1.0.0.dev1-cp38-cp38-manylinux1_x86_64.whl
+RUN pip3 install --no-cache-dir zimscraperlib-1.4.0.dev1-py3-none-any.whl && rm zimscraperlib-1.4.0.dev1-py3-none-any.whl
+
+COPY requirements.txt /src/
+RUN pip3 install --no-cache-dir -r /src/requirements.txt
+COPY wikihow2zim /src/wikihow2zim
+COPY setup.py *.md /src/
+RUN cd /src/ && python3 ./setup.py install
```
Comment (rgaudin) [inline Dockerfile]: /src can be removed after install. /output can be created at that moment as well

**Review by rgaudin — CHANGES_REQUESTED** [review]
Looks good ; please take a look at my inline comments.

**Comment by ElenaKusevska** [comment]
@rgaudin Please review the PR again, and let me know if it's OK, or I should make some more changes.

**Comment by rgaudin** [comment]
Thank you @ElenaKusevska ; that's good. Can you please rewrite the git history? Then we'll merge.

Also, please remove the `WORKDIR /output`, doesn't make much sense.

**Review by rgaudin — APPROVED** [review]

**Comment by ElenaKusevska** [comment]
@rgaudin check if it's OK now.

**Comment by rgaudin** [comment]
That will do. thanks

**Pull request merged**